src/Umbraco.Core/Services/IMembershipRoleService.cs

```diff
@@ -9,6 +10,7 @@ public interface IMembershipRoleService<out T>
     {
         void AddRole(string roleName);
         IEnumerable<string> GetAllRoles();
+        IEnumerable<IMemberGroup> GetAllRolesTyped();
```

Comment:
I was thinking of removing it, but was wondering if I'd impact something I didn't know about. I'll see if I can switch to just this typed one and what it impacts.

Reply:
Removed in recent push - along with some formatting as per style rules.

(NB, I've also just corrected a bunch of namespaces and this will look like more files changed than I've actually changed for members functionality)